src/components/UI/MainWalletButton/MainWalletButton.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {useColors} from '../../../hooks';
+import {toClasses} from '../../../utils';
+import {Button} from '../index';
+import styles from './MainWalletButton.module.scss';
+
+export const MainWalletButton = ({text, onClick, ...props}) => {
```

Comment:
**Transfer**Button is a **Wallet**Button?

Reply:
good point, I'll fix this